pkg/cmd/roachtest/roachtestflags/flags.go

```diff
@@ -357,6 +357,12 @@ var (
 		Usage: `Use SpotVM to run tests, If the provider does not support spotVM, it will be ignored`,
 	})
 
+	AutoKillThreshold float64 = 1.0
+	_                         = registerRunFlag(&AutoKillThreshold, FlagInfo{
+		Name:  "auto-kill-threshold",
+		Usage: `The failure rate threshold at which all remaining tests will be automatically terminated.`,
```

Comment:
Nit: `Percentage of failed tests before all remaining tests are automatically terminated.`

Reply:
Done